crates/llm-ls/src/backend.rs

```diff
@@ -172,6 +203,15 @@ pub(crate) fn build_body(
             request_body.insert("model".to_owned(), Value::String(model));
             request_body.insert("stream".to_owned(), Value::Bool(false));
         }
+        Backend::LlamaCpp { .. } => {
+            if model.to_lowercase().contains("deepseek") {
+                request_body.insert("prompt".to_owned(), Value::String(format!("\"{}\"", prompt)));
+            } else {
```

Comment:
Why is this necessary?

Reply:
@McPatate When I was doing tests I used deepseek as the base model. Its payload required the prompt to be between quotes, so that's the reason.